docs/upload_files.md

```diff
@@ -41,8 +41,8 @@ For more examples, see [CLI examples](/examples/cli/README.md) and [SDK examples
 
 # Metadata
 
-To add metadata to your files, create one metadata file for each TXT or PDF file you upload. The metadata file must be a JSON with the same name as the file whose metadata it contains and the extension `meta.json`.
+To add metadata to your files, create one metadata file for each file you upload. The metadata file must be a JSON with the same name as the file whose metadata it contains and the extension `meta.json`.
```

Comment:
@agnieszka-m We're enabling uploading additional file types (e.g. csv, docx etc.) as specified in the variable `ALLOWED_TYPE_SUFFIXES`. I removed here in the user docs specific file types since we expect them to change in the future. Are you fine with this?

Reply:
BTW. One advantage of this setup is that while generating the docs we could theoretically read the variable of allowed file types and render this to this markdown. (Out of scope for now)